src/PrimitiveObsession/IoC/EngineContainerBuilder.cs

```diff
@@ -17,7 +17,9 @@ public IContainer Build()
         {
             var builder = new ContainerBuilder();
 
-            builder.RegisterModule(new PrimitiveObsessionModule(_config));
```

Comment:
Qui siamo iniettando un service locator dentro il modulo, il che è un po' ironico, se pensiamo che AutoFac serve anche ad evitare l'uso dei Service Locator.

Trovo molto più corretto iniettare direttamente i 2 parametri. 


In questo commit c'è ancora un Service Locator in `EngineContainerBuilder`. Quando tra qualche commit eliminiamo anche quel Service Locator,  la classe si semplificherà abbastanza da poterci permettere di eliminarla, rimuovendo un layer.